doc/_templates/customindex.html

```diff
@@ -31,21 +31,26 @@ <h2>Decentralized</h2>
       <div class="body-masthead-copy">
         <h2>Open Standard</h2>
         <p>
-          Autocrypt is an open standard for e-mail encryption.
+          Autocrypt is an open specification for e-mail encryption.
           Under the hood, it uses OpenPGP and other e-mail standards.
         </p>
       </div>
     </div>
   </div>
   <div>
-    <h2>Our Mission</h2>
+    <h2>What is Autocrypt? What is its status?</h2>
     <p>
-      End-to-end encrypted e-mail has been around for decades, but has failed to see wide adoption outside of specialist communities.  Autocrypt aims to provide encryption convenient enough for much wider adoption. Through strengthening the e-mail ecosystem step-by-step, we want to increase user autonomy and privacy for everyone.
-    </p>
-    <p>
-      The Autocrypt effort is driven by a diverse group of mail app developers, hackers and researchers who are willing to take fresh approaches, learn from past mistakes, and collectively aim to increase the overall end-to-end encryption of e-mail in the net.
+      Autocrypt is a specification which mail application programmers can implement
+      to offer their users convenient end-to-end encryption for e-mail. The first specification
+      has been released in December 2017 and a few mail programs already have or are heading 
+      towards Autocrypt-capable releases in spring 2018.  At this point, we are reaching out 
+      to programmers, interested users and researchers to join us in testing and refining 
+      the "Level 1" specification which provides the baseline for e-mail encryption.  Feedback and 
+      questions are welcome, see the <a href="{{ pathto('contact') }}">Contact Page</a>.
```

Comment:
There is a lot of information in this paragraph.  Arguably, it doesn't even have a single topic.  Would it be OK to break it into two paragraphs?